### PR TITLE
bump thanos image version

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>
Bump thanos image version.

### CODE REVIEW

**Comment by metalmatze** [comment]
Nothing in this Repo should need updates for Thanos between these versions.

File: jsonnet/kube-prometheus/kube-prometheus-thanos-sidecar.libsonnet [hunk 1]
```diff
@@ -5,10 +5,10 @@ local servicePort = k.core.v1.service.mixin.spec.portsType;
 {
   _config+:: {
     versions+:: {
-      thanos: 'v0.5.0',
+      thanos: 'v0.7.0',
     },
     imageRepos+:: {
-      thanos: 'improbable/thanos',
+      thanos: quay.io/thanos/thanos',
```
Comment (peterbueschel) [inline jsonnet/kube-prometheus/kube-prometheus-thanos-sidecar.libsonnet]: There is a missing single quote, if my eyes are correct.

Comment (LiliC) [reply jsonnet/kube-prometheus/kube-prometheus-thanos-sidecar.libsonnet]: @peterbueschel nicely spotted, do you want to open a PR to fix this?

Comment (yeya24) [reply jsonnet/kube-prometheus/kube-prometheus-thanos-sidecar.libsonnet]: Oh, sorry for that miss. I will open a PR to fix

**Pull request merged**